src/debian/01_password

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/sh
+
+GRUB_PASS=
+
+echo superusers="root"
+echo password_pbkdf2 root $GRUB_PASS
+awk "/^menuentry 'Debian/,/}/" /boot/grub/grub.cfg | awk '{gsub(/menuentry.*{/,"menuentry 'Debian' --unrestricted {"); print}'
```

Comment:
Could probably be inside a single command:
```
awk '/^menuentry 'Debian/{gsub(/menuentry.*{/,"menuentry 'Debian' --unrestricted {"); print}'
```

Reply:
Add a comment showing before/after command results since it's hard to decypher.
bad menuentry { --> new menuentry {